contrib/kube-prometheus/hack/example-service-monitoring/deploy

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-kubectl apply -f manifests/examples/example-app
+kubectl apply -f experimental/examples/example-app
```

Comment:
As far as I can tell, this path does not exist. Don't you mean `examples/example-app` here? Same below in `teardown`?

Reply:
good catch!